l1-contracts/test/foundry/unit/concrete/Bridges/L1SharedBridge/_L1SharedBridge_Shared.t.sol

```diff
@@ -123,16 +133,15 @@ contract L1SharedBridgeTest is Test {
             abi.encodeWithSelector(L1SharedBridge.initialize.selector, owner)
         );
         sharedBridge = L1SharedBridge(payable(sharedBridgeProxy));
+
+        l1Erc20Bridge = new L1ERC20Bridge(sharedBridge);
+        l1ERC20BridgeAddress = address(l1Erc20Bridge);
+
         vm.prank(owner);
         sharedBridge.setL1Erc20Bridge(l1ERC20BridgeAddress);
         vm.prank(owner);
-        sharedBridge.setEraPostDiamondUpgradeFirstBatch(eraPostUpgradeFirstBatch);
```

Comment:
I remember there was a problem with these setters. Does removing them solve it because this change creates a lot of repetitive code?

Reply:
just in finalize withdrawal and I've wanted to test it. I don't think i creates too much repetitive code. There is also this case of pre-upgrade and post-upgrade function calls.